input/fsh/omrs-related-person.fsh

```diff
@@ -0,0 +1,137 @@
+Profile: OMRSRelatedPerson
+Parent: RelatedPerson
+Id: omrs-related-person
+Title: "OpenMRS Related Person"
+Description: "A FHIR RelatedPerson as understood by OpenMRS Relationship"
+* id 1..1
+* patient 1..1
+* identifier 1..1
+* address 0..*
+* name 0..*
+* gender 0..1
+* birthDate 0..1
+* active 1..1
+* period 1..1
```

Comment:
Work on the cardinalities, Please look through this https://build.fhir.org/relatedperson.html It should be able to guide you.

Reply:
Updated to match the official FHIR IG